test.js

```diff
@@ -14,3 +14,10 @@ test('--regex', async t => {
 	await execa('./cli.js', ['--regex=\\bb.*?\\b', '--replacement=foo', filePath]);
 	t.is(fs.readFileSync(filePath, 'utf8'), 'foo foo foo');
 });
+
+test('test new line', async t => {
+	const filePath = await tempWrite(',');
```

Comment:
Perhaps make the string something like a,b,c instead?

Reply:
Edited the test to replace ',' in the string a,b,c.